src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/DataNodeController.java

```diff
@@ -53,14 +52,18 @@ public DataNodeController(BaseAnalysisService<Analysis> analysisService,
     }
 
     @Override
-    protected DataNode convertRegisterDtoToDataNode(CommonDataNodeRegisterDTO commonDataNodeRegisterDTO) {
-
-        return conversionService.convert(commonDataNodeRegisterDTO, DataNode.class);
+    protected DataNode createEmptyDataNode() {
```

Comment:
may be, build? because two "create"s in a row (createEmptyDataNode and then dataNodeService.create) look a bit confusing